### PR TITLE
Fixed translation of shortcuts in Preferences

### PR DESCRIPTION
**Pull request by tsujan**
Also, worked around the Qt's bug about the Meta key by preventing multiple shortcuts.

Fixes https://github.com/lxqt/qterminal/issues/731

NOTE:

Actually, entering multiple shortcuts into `QKeySequenceEdit` can never result in multiple shortcuts but just creates a complex shortcut, all of whose elements should be pressed in a sequence. That's a small design flaw we have to tolerate.

That being said, (1) multiple shortcuts for a single action aren't very useful either because Qt only shows the first one in the main GUI, and (2) The separator `|` for multiple shortcuts isn't removed from the code because it can be used by editing the config file manually.

### CODE REVIEW

**Comment by tsujan** [comment]
@stefonarch, please test it when you find the time. All actions and they shortcuts should appear in the GUI language inside the shortcut editor or on menus, not necessarily in English. Also, shortcuts should be modified without problem. Here, it worked with French.

**Comment by stefonarch** [comment]
As for the column it looks fine, but the shortcuts are in still in english (example "shift" instead of "maiusc" in italian) but I'm not sure if those are translated anywhere.

I'd ship it, if there are problems: point release...

**Comment by tsujan** [comment]
> example "shift" instead of "maiusc" in italian

I'm sure that's how Qt handles `Shift` in Italian — right or wrong — because, for example, I see `Shift+Destra` and `Shift+Sinistra` when I run QTerminal+patch with `LANGUAGE=it qterminal` (I mean "Destra" and "Sinistra").

I've seen a similar Qt problem about Dutch but with Right/Left.

> I'd ship it, if there are problems: point release...

Let's wait for @yan12125; he's the master of QTerminal.

**Comment by yan12125** [comment]
Could this be postponed to the next version? In my memory the part for loading translations is fragile and I prefer to properly review it before merging.

**Comment by stefonarch** [comment]
> I'm sure that's how Qt handles `Shift` in Italian

Yeah, didn't look properly, saw just "shift" and didn't look further.

**Comment by tsujan** [comment]
> Could this be postponed to the next version?

I have no problem with that.

> In my memory the part for loading translations is fragile

I didn't touch it; I just added Qt translations, in the way all LXQt components do it.

File: src/propertiesdialog.cpp
Comment (yan12125) [inline src/propertiesdialog.cpp:443]: Could you extract these lines to a function? The complex regular expression may be changed in the future and I'd like to manage them in one place.

**Review by yan12125 — APPROVED** [review]
> I didn't touch it; I just added Qt translations, in the way all LXQt components do it.

Yep it turns out I worried too much. I have some bad experiences when testing translations across different platforms (macOS & Linux), but this pull request looks platform-independent when I checked again.

Looks good overall. Just a minor coding-style stuff.

File: src/propertiesdialog.cpp
Comment (tsujan) [reply src/propertiesdialog.cpp:443]: Will do it.

**Comment by tsujan** [comment]
> I have some bad experiences when testing translations across different platforms (macOS & Linux)

If shortcuts aren't correct in macOS, please tell me! They should be correct because they're saved in the portable format and shown in the native format but who knows what might happen with a complex code?

**Comment by yan12125** [comment]
Ah, that reminds me of https://github.com/lxqt/qterminal/issues/570. I guess that issue will also be fixed after this pull request? I don't have access macOS now and thus I cannot check.

**Comment by tsujan** [comment]
> Ah, that reminds me of #570.

Yes, that's what I meant.

> I guess that issue will also be fixed after this pull request?

In the case of FeatherPad, for example, after I respected the above-mentioned format rules, macOS issues were fixed. Here, the code is inherited from the old times and is a little weird; so, I can't be sure. I don't have macOS; if you do, would you please test and tell me?

**Comment by tsujan** [comment]
> I don't have access macOS 

Oh, sorry, I missed this part of your comment.

**Comment by tsujan** [comment]
@yan12125 
In [a separate commit](https://github.com/lxqt/qterminal/pull/732/commits/527b19fcd6b2227ed3c4391ae6bd4a884cacb4ec), I added a static method to `Properties` for removing accelerators — didn't find a better place for it. If you had/have another place in mind, please tell me!

**Comment by yan12125** [comment]
Thanks! A static function in Properties looks good.

**Pull request merged**
